Utilities/ScanSubscription.ps1

```diff
@@ -0,0 +1,362 @@
+# Description: This script is used for chercking enabled VM sizes and VM core quota in the given azure subscription.
+# It will upload subscription id and unavailable VM/insufficient VM core to db
+# This script will be executed by 'Penguinator-Validate-Azure-Subscription-Info' pipeline
+# This script should be under .\lisav2\Utilities directory
+
+$clientSecret = "$(CLIENT_SECRET)"
+$clientID = $env:CLIENT_ID
+$tenantID = $env:TENANT_ID
+$subID = $env:SUBSCRIPTION_ID
```

Comment:
Can you pass as the parameters of this script?

Reply:
fixed